frontend/src/sub-components/CardOptions.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+class CardOptions extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+    render() {
+        const Option = styled.div`
```

Comment:
These should be defined outside the render method.

Reply:
done.